tests/virt_autotest/virsh_external_snapshot.pm

```diff
@@ -51,6 +51,10 @@ sub run_test {
     my @vm_hostnames_inactive_array = split(/\n+/, $vm_hostnames_inactive);
 
     foreach my $guest (keys %xen::guests) {
+        my $guest_disk_type = script_output("virsh dumpxml $guest | grep '<driver ' | grep -o "type='.*'" | cut -d "'" -f2");
+        record_info "INFO", "SKIP Snapshot test if guest disk type as raw";
```

Comment:
better to print skipping test after we determine it is 'raw' than before we know it. what do you think?

Reply:
Great, agree on this. 
I had updated code already. Thank you so much for your great help.